src/CONST.ts

```diff
@@ -814,6 +814,7 @@ const CONST = {
         MAX_PENDING_TIME_MS: 10 * 1000,
         MAX_REQUEST_RETRIES: 10,
     },
+    WEEK_STARTS_ON: 1, // Monday
```

Comment:
I feel like `WEEK_START_DAY` sounds more correct (its a more natural language vs tech/dev speak), especially because we have the function below:

```js
function getWeekStartsOn(): WeekDay {
    return CONST.WEEK_STARTS_ON;
}
```

Reply:
NAB, WEEK_START_DAY reads better as constant